server/views/pages/goal/add-note/index.njk

```diff
@@ -4,7 +4,23 @@
 {% set pageTitle = applicationName + " - Home" %}
 
 {% block beforeContent %}
-  {% include "../../../partials/breadCrumb.njk" %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: dpsUrl
+      },
+      {
+        text: "Learning and work progress",
+        href: "/"
+      },
+      {
+        text: "Manage " + prisonerSummary.firstName | capitalize + " " + prisonerSummary.lastName | capitalize + "'s learning plan",
+        href: "/"
```

Comment:
Minor - should this href be linking to `/` ? or should it be linking to the prisoner overview page as per your other changes?

Reply:
Good spot resolved